custom_components/healthchecksio/config_flow.py

```
@@ -1,142 +1,220 @@
                     self._errors["base"] = "auth"
 
-        return await self._show_initial_config_form(user_input)
+        DATA_SCHEMA = vol.Schema(
+            {
+                vol.Required(
+                    CONF_API_KEY,
+                    default=user_input.get(CONF_API_KEY)
+                    if user_input is not None
+                    else None,
+                ): str,
+            }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)